misc/version.sh

```diff
@@ -34,9 +34,11 @@ DEPS=" ${ARCH}"
 if test -f ${SRCDIR}/check-deps/have_libdw; then
     DEPS="${DEPS} dwarf"
 fi
-if test -f ${SRCDIR}/check-deps/have_libpython2.7 ||
-   test -f ${SRCDIR}/check-deps/have_libpython3; then
-    DEPS="${DEPS} python"
+if test -f ${SRCDIR}/check-deps/have_libpython2.7; then
+    DEPS="${DEPS} python2.7"
+fi
+if test -f ${SRCDIR}/check-deps/have_libpython3; then
+    DEPS="${DEPS} python3"
 fi
```

Comment:
uftrace uses python3 only when it has both versions so this should be as follows.
```suggestion
if test -f ${SRCDIR}/check-deps/have_libpython3; then
    DEPS="${DEPS} python3"
elif test -f ${SRCDIR}/check-deps/have_libpython2.7; then
    DEPS="${DEPS} python2.7"
fi
```

Reply:
I will accept the suggestion.
Thank you!

Reply:
As we all know it's python 2.7, I'd rather have it just like "python2" instead of "python2.7".

Reply:
I will change to "python2" !